src/settings.py

```diff
@@ -2,24 +2,34 @@
 
 
 from typing import Dict
-from pydantic import BaseSettings, HttpUrl
+from pydantic import BaseSettings, HttpUrl, AnyHttpUrl
 
 
 class AggregatorSettings(BaseSettings):
     """Create Settings from env."""
```

Comment:
Do these need to be classes?

Reply:
Yes, look at https://pydantic-docs.helpmanual.io/usage/settings/